### PR TITLE
feat: Added skeleton loaders to Credentials Page

### LINKED ISSUE #943: Add credential and credential details skeleton loaders

### LINKED ISSUE #1352: Implement loading skeleton for vaults filter flyout
image

### PR DESCRIPTION
**Pull request by HeidiHan0000**
- Added loading states to the Vaults page and the Credentials page according to designs found [here](https://app.abstract.com/projects/9ec92568-60a2-4fee-8215-5aa997d52428/branches/dc10bcee-d9a1-44f9-b644-8f843c6f1a19/commits/68d43e7308faabf1b365be94685d85413c7ad001/files/95d5757f-f10e-43f6-ad50-0faa236217fd/layers/EE056E96-973D-4FE0-BA53-77C250F7F8E1?collectionId=84dc8bfa-58a9-4a7b-a4a9-0bb25852746b&collectionLayerId=d01add70-5db3-4995-b0e7-2bcfb9051dd6&mode=build)
- Closes #943 
- Closes #1352 

https://user-images.githubusercontent.com/44453261/175665759-ac8bd813-0514-4f2c-b10b-80ba32d567cb.mov

\
\
Vaults Page:

https://user-images.githubusercontent.com/44453261/175573310-e3c4e4ae-da13-4cc4-b130-deffb85c2b62.mov

https://user-images.githubusercontent.com/44453261/175573577-e5f71611-d8df-4eb6-a33d-72bc8aea29a1.mov

\
\
Credentials Page:

https://user-images.githubusercontent.com/44453261/175574094-833cb92c-423b-4108-b962-676b80b363ec.mov

https://user-images.githubusercontent.com/44453261/175573901-95769580-b278-4435-8d17-246ee82a8c7c.mov

Signed-off-by: heidihan0000 <daeun.han@avast.com>

### CODE REVIEW

**Comment by birtony** [comment]
Nice progress! There is one thing I noticed on the first video - the loading state seems to turn off a bit too early. You can see that the default vault shows up right after the skeletons disappear, but the rest of the vaults still take time to show up. Could you investigate this, please? @HeidiHan0000

File: cmd/wallet-web/src/components/SkeletonLoader/CredentialPreviewSkeletonComponent.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div class="flex flex-col justify-start items-start px-0 w-full">
+    <div
+      class="flex z-10 flex-row justify-between items-center lg:py-6 pr-8 pl-5 w-full lg:h-24 bg-neutrals-whiteLilac rounded-xl"
+      :class="isTall ? 'h-24 py-6' : 'h-20 py-5'"
```
Comment (talwinder50) [inline cmd/wallet-web/src/components/SkeletonLoader/CredentialPreviewSkeletonComponent.vue:11]: where is this isTall defined ?

Comment (HeidiHan0000) [reply cmd/wallet-web/src/components/SkeletonLoader/CredentialPreviewSkeletonComponent.vue:11]: I defined it as a prop, set to `false` by default and passed in `true` for the Credential Details Page. On the Credential Details Page, `isTall` is `true` for any screen size. On the Credentials Page, `isTall` is only `true` for large screens (this is based on the designs)

**Review by birtony — APPROVED** [review]

**Comment by HeidiHan0000** [comment]
Comment from Mike:
"Lots of loaders going on for data that hasn’t changed, this causes a very clunky and “slow” feeling experience.
Data that hasn't changed (vault cards, credential cards...) should be cached and should be quick to load."

Created #1746 to address this

**Pull request merged**